src/en/page-templates/basic/basic.md

```diff
@@ -0,0 +1,82 @@
+tags: ["templates"]
+date: "git Last Modified"
+github: https://github.com/cds-snc/gcds-examples/blob/feat/add-basic-page-templates/templates/english/basic-page-template.html
+figma: https://figma.com
+---
+
+# Basic page template
+
+The Basic Page template provides accessibility essentials for page structure and heading hierarchy. Brand elements, like the header and footer, make it recognizable as a Government of Canada page.
+
+## When to use
+
+For use on Canada.ca and other GC sites.
+The basic page template is for non-specialized pages across Canada.ca and is an optional template. It has the starting elements needed for a simple page and those required for most GC pages. You can use it as a canvas to add or combine <gcds-link href="{{ links.components }}">components</gcds-link> from the GC Design System.
+
+## Preview
+
+<gcds-button class="md:d-inline-block d-block md:me-400 me-0 md:mb-0 mb-400" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicPreview }}" target="_blank">Open demo in new tab</gcds-button>
+<gcds-button class="md:d-inline-block d-block" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicCode }}" target="_blank">Get code in new tab</gcds-button>
+
+<img class="max-width-content b-sm b-default mb-400 p-400" src="/images/en/templates/basic-page-preview.png" alt="A basic page template includes the Government of Canada header and footer, H1, H2 and H3 heading sections with text underneath each section."/>
+
+## How to implement
+
+To implement page templates into your project or to prototype in HTML, download the specific template you need in GitHub:
+
+- <gcds-link external href="{{ links.pageTemplatesBasicGithubLink }}">Basic page template</gcds-link>
+- <gcds-link external href="{{ links.pageTemplatesBasicExtOTPGithubLink }}">Basic page table of contents template</gcds-link>
+
+To prototype in Figma, find the <gcds-link external href="{{ links.pageTemplatesBasicFigma }}">Basic Page Template</gcds-link> in the GC Design System library.
+
+## Design considerations
+
+### Improve navigation on longer pages
+
+Include an "On this page" section with bulleted anchor links for pages with four or more sections. Anchor links  to improve navigation and help people find relevant content.
+
+### Preview "On this page"
+
+<gcds-button class="md:d-inline-block d-block md:me-400 me-0 md:mb-0 mb-400" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicExtOTPPreview }}" target="_blank">Open demo in new tab</gcds-button>
+<gcds-button  button-role="secondary" type="link" href="{{ links.pageTemplatesBasicExtOTPCode }}" target="_blank">Get code in new tab</gcds-button>
+
+<img class="max-width-content b-sm b-default mb-400 p-400" src="/images/en/templates/basic-page-on-this-page-preview.png" alt='A basic page template with a "On this page" section includes the Government of Canada header and footer, a "On this page" section with three sections listed underneath.'/>
```

Comment:
Same comment: `mb-400` and `p-400` need to be changed to 300 please to match the new spacing classes.

Reply:
```suggestion
<img class="max-width-content b-sm b-default mb-300 p-300" src="/images/en/templates/basic-page-on-this-page-preview.png" alt='A basic page template with a "On this page" section includes the Government of Canada header and footer, a "On this page" section with three sections listed underneath.'/>
```

Reply:
Done!